src/pages/EmergencyViewPage/EmergencyViewPage.scss

```
@@ -20,23 +20,24 @@ $tab-item-room-width: 180px;
   flex-direction: column;
   z-index: z(emergency-content);
   height: 100%;
+  position: absolute;
+  height: calc(100vh - #{$navbar-height});
```

Is the footer supposed to be visible as well? In that case, can use something like https://github.com/sparkletown/sparkle/blob/env/burn-staging/src/scss/constants.scss#L74

```suggestion
  height: calc(100vh - #{$reserved-page-height});
```

---

fixed 308745f